config/routes.rb

```diff
@@ -11,6 +11,8 @@
   get "sources" => redirect('/quellen')
   resources :sources, path: 'quellen', only: [:index, :show]
 
+  resources :submit_source, path: 'submit_source', only: [:new, :create]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.